doc/tutorials/workshop/content/exercises/12-common-rules.md

```diff
@@ -0,0 +1,98 @@
+---
+Title: All types of rules
+PrevPage: 11-node-rules
+NextPage: ../finish
+---
+Common types of rules
+===================
+
+The rules we created in the previous chapters leveraged the `yamlfile_value`,
+`file_permissions` and `sysctl` template from CaC/content project. There are
+many more templates available in the project and each one enable us to quickly
+create rules with a specific behavior.
+
+In this section we will go through the most common types of checks when
+assessing the security posture of a cluster and the templates used to
+implement them.
+
+## Platform checks
+
+### Checking Kubernetes resources
+
+Checking the configuration of a Kubernetes resource is the obvious task when
+one wants to check the cluster's security posture.
+This type of check is handled by the [yamlfile_value](https://complianceascode.readthedocs.io/en/latest/templates/template_reference.html#yamlfile-value)
+template.
+
+You can write a rule and make use of the template by yourself, or
+use the `./utils/add_platform_rule.py` script showcased in past sections.
+Note that more advanced uses of the template will require you to write the input data by yourself.
+
+One example of this type of rule is a checking whether any registry configured
+for import allow use of insecure protocols. The rule checks the Cluster API and
+assess whether any `allowedRegistriesForImport` has `'{"insecure": true}':
+[ocp_insecure_allowed_registries_for_import](https://github.com/ComplianceAsCode/content/blob/master/applications/openshift/registry/ocp_insecure_allowed_registries_for_import/rule.yml)
+
+Another example is checking whether RBAC roles are defined. 
+[rbac_roles_defined](https://github.com/ComplianceAsCode/content/blob/master/applications/openshift/rbac/rbac_roles_defined/rule.yml)
+
+## Node checks
+
+### Checking for a KubeletConfig setting
+
+With Kubelet being the primary agent on the node, its configuration becomes
+important to assess.
+
+You can also use the [yamlfile_value](https://complianceascode.readthedocs.io/en/latest/templates/template_reference.html#yamlfile-value)
+template for this. Note though that these checks are Node checks, not Platform checks.
```

Comment:
Thanks for calling this out.

Someone might still ask "why would this be a Node check if it's evaluating the Platform?"

I wonder if we need to elaborate on that above where we can say something like:

_Platform rules are only invoked once, making them idea for evaluating the state of the platform. For example, checking the value of a particular Kubernetes resource, or cluster configuration option. Node rules may also evaluate the configuration of the platform, but they're intended to be run on multiple nodes. For example, a node rule can evaluate etcd configuration across all OpenShift master nodes. A node rule may also check the permissions of `/etc/sshd_config`, which isn't specific to OpenShift._ 

I'm trying to prevent people from automatically assuming all rules that check Kubernetes-like things will be platform rules.

What do you think?

Reply:
Thanks, this is a great question.

Based on the suggested text I added a few sentences right at the beginning of the Platform and Check sections.
Please take a look.